deploy/helm/templates/node-collector-daemon-set.yaml

```diff
@@ -129,7 +133,55 @@ spec:
               readOnly: true
             - name: logcheckpoints
               mountPath: {{ .Values.otel.logs.filestorage.directory }}
+{{- end }}
+{{- if and .Values.ebpfNetworkMonitoring.enabled .Values.ebpfNetworkMonitoring.kernelCollector.enabled }}
+        - name: swi-kernel-collector
+          image: "otel/opentelemetry-ebpf-kernel-collector:v0.10.0"
```

Comment:
Should we make this a variable?

Reply:
I wasn't sure if there is use case for it, but yeah for better flexibility I will add option to `values.yaml` to be able to override images

Reply:
fixed